File: src/app-server/csv-init.js
function csv_init()
{
 const csv=obj() 
 
 csv.tmp="tmp"
 
 csv_deinit(csv)
 dir_make(csv.tmp)
  
 const dir=dir_current()

 dir_change(csv.tmp)
 os_spawn("sudo","python3","../src/app-server/res/csv-save.py")
 dir_change(dir)
 
 return csv
}

File: src/app-server-jneeg/csv-init.cs
gn csv_init
 let csv obj
 let tmp path_tmp "csv"

 assign csv.tmp tmp

 csv_deinit csv
 dir_make csv.tmp

 let dir dir_current
 let path "src/app-server-jneeg/res/csv-save.py"
 let path path_real path

 dir_change csv.tmp
 run os_detach "sudo" "python3" path
 dir_change dir

 ret csv
end
